### ISSUE TITLE
[BUG?] Increasing load due blocked/dead processes

### ISSUE THREAD

**Issue by wursterje** [longhorn/longhorn#5441]
Not sure if this is related due Longhorn or Kubernetes but we experience theses issues in different environments, one of them is documented here:

It's maybe related if a pod is terminated and a volume detached.

15 min Load:

![system_load15](https://user-images.githubusercontent.com/14123012/220585386-f79601be-db2d-415b-86dc-6c2fc327192b.png)

Processes Dead/Locked:

![blocked_processes](https://user-images.githubusercontent.com/14123012/220585644-abd6de76-8472-4385-af0f-99a233ce7823.png)

`Longhorn Version: 1.4.0`

```
lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

```
uname -a
Linux theia 5.10.0-20-amd64 #1 SMP Debian 5.10.158-2 (2022-12-13) x86_64 GNU/Linux
```

```
k3s --version
k3s version v1.24.9+k3s1 (4dcf0a33)
go version go1.18.9
```

Processes:  
[processes.log](https://github.com/longhorn/longhorn/files/10802374/processes.log)

**Comment by derekbit** [comment]
A ReadWriteMany volume? If yes, it's fixed in this [issue](https://github.com/longhorn/longhorn/issues/5296) and will be included in the upcoming v1.4.1.

**Comment by wursterje** [comment]
@derekbit Yes, its RWX. Workaround with umount -f did its job.
